server/lib/src/constants/entries.rs

```diff
@@ -642,623 +643,526 @@ impl EntryClass {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
 /// Built-in group definitions
 pub struct BuiltinGroup {
     pub name: &'static str,
     description: &'static str,
-    classes: Vec<EntryClass>,
     uuid: uuid::Uuid,
-    member: uuid::Uuid,
+    members: Vec<uuid::Uuid>,
+    dyngroup: bool,
+    dyngroup_filter: Option<Filter>,
+    // TODO: additional attributes (for things like the uihint group)
+    extra_attributes: Vec<(Attribute, Value)>,
```

Comment:
Do we have other extra attrs beside uihints? Could we just list uihints as an option here?

Reply:
I don't know (yet) but I figured it would be easier at this stage of the yak-shaving

Reply:
Fair :)